### ISSUE TITLE
Some files in zig binaries are missing 

### ISSUE THREAD

**Issue by Rocknest** [ziglang/zig#3431]
I checked both linux and windows binaries and they have empty `std/json/*` and `std/unicode/*`

**Comment by andrewrk** [comment]
This is actually ok, it is avoiding installing test files:

 * https://github.com/ziglang/zig/tree/6082c147533cbefe9f27d8f32b6485c8c43a738f/lib/std/json
 * https://github.com/ziglang/zig/tree/6082c147533cbefe9f27d8f32b6485c8c43a738f/lib/std/unicode

The issue here is actually that it is not *omitting* an empty directory.

**Comment by Rocknest** [comment]
Now this files may be useful (running std tests to generate docs)

**Comment by andrewrk** [comment]
I agree that we either need to ship pre-built std lib docs or the test files. The compiler_rt tests, in particular, total 11.7 MiB, which is greater than the size of the pre-built std lib docs. Since the process for building std lib docs will be a bit complicated, and take some time, I think I will prefer to ship std lib docs for now.

For working on doc generation, as you are doing in #3429, it is reasonable to require a source checkout.
